### PR TITLE
Switch to pnpm, add CONTRIBUTING.md and CI check on PRs

### PR DESCRIPTION
**Pull request by nathanaelhoun**
- [x] Switch to `pnpm` package manager to be consistent with other repositories (removed `package-lock.json` and `yarn.lock`).
- [x] Added `CONTRIBUTING.md` and referenced it into the README.
- [x] Added nextjs ci check on PRs
- [x] Cleanup unused files

### CODE REVIEW

File: .github/workflows/nextjs.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  check-next-website:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        node-version: [16.x]
+
+    steps:
+      - uses: pnpm/action-setup@v2.1.0
+        with:
+          version: 6.0.2
```
Comment (vraravam) [inline .github/workflows/nextjs.yml:17]: The latest version of `pnpm` is 7.1.0 - lets move to that if possible.

File: .github/workflows/nextjs.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+name: NextJS check
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  check-next-website:
+    runs-on: ubuntu-20.04
```
Comment (vraravam) [inline .github/workflows/nextjs.yml:9]: Do we want to upgrade to ubuntu 22.04?

File: .github/workflows/publish-github-pages.yml [hunk 3]
```diff
@@ -9,10 +9,15 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
+      - uses: pnpm/action-setup@v2.1.0
+        with:
+          version: 6.0.2
```
Comment (vraravam) [inline .github/workflows/publish-github-pages.yml:14]: Same here for pnpm version

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,22 +1,79 @@
...
+	- [Install dependencies](#install-dependencies)
+	- [Add your modifications](#add-your-modifications)
+	- [Build the website locally](#build-the-website-locally)
+- [Learn More](#learn-more)
+
+<!-- /TOC -->
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [Ferdium Code of Conduct](https://raw.githubusercontent.com/ferdium/ferdium-app/develop/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code.
```
Comment (vraravam) [inline CONTRIBUTING.md:25]: Should we point to the code of conduct from the other repo, or duplicate that file in this one?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,22 +1,79 @@
...
+Please report unacceptable behavior to [hello@ferdium.org](mailto:hello@ferdium.org).
+
+## How do I contribute?
+
+### Setup your development environment
+
+#### Node.js, pnpm
+
+You will need a working installation of Node.js, with the `pnpm` package manager.
+It is advised to use the version recommended in the [`ferdium-app` repository](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md#nodejs-npm-pnpm).
```
Comment (vraravam) [inline CONTRIBUTING.md:35]: Same here - the 2 repos might evolve independently of each other (though, as maintainers we would try to keep them in sync as to node/npm/pnpm versions). Is that ok or do we want to keep the documentation also independent?

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,22 +1,79 @@
...
+
+_Note:_ You can choose any version manager to manage multiple versions of `node` and `npm`. For eg, [nvm](https://github.com/nvm-sh/nvm) or [asdf](https://github.com/asdf-vm/asdf).
+
+#### Git
+
+The version [2.23.0](https://github.com/git-for-windows/git/releases/tag/v2.23.0.windows.1) for Git is working fine for development. You can then use the console from Git to do the development procedure.
```
Comment (vraravam) [inline CONTRIBUTING.md:41]: I would much rather point to the main git repo/url, and not to the clone for Windows.
https://git-scm.com/

File: pages/faq.tsx [hunk 7]
```diff
@@ -107,7 +107,7 @@ function FAQ({}: Props) {
         </FAQEntry>
         <FAQEntry title="I can't to open Ferdium after downloading. What should I do?">
           <>
-            This is because the releases aren't yet signed. We are still working on getting an Apple developer license and a certificate for Windows.
+            This is because the releases aren&apost yet signed. We are still working on getting an Apple developer license and a certificate for Windows.
```
Comment (vraravam) [inline pages/faq.tsx:110]: Is this change necessary?

**Review by vraravam — CHANGES_REQUESTED** [review]

File: .github/workflows/publish-github-pages.yml (hunk 3 above)
Comment (nathanaelhoun) [reply .github/workflows/publish-github-pages.yml:14]:
```suggestion
          version: 7.1.0
```

File: .github/workflows/nextjs.yml (hunk 2 above)
Comment (nathanaelhoun) [reply .github/workflows/nextjs.yml:9]:
```suggestion
    runs-on: ubuntu-22.04
```

File: CONTRIBUTING.md (hunk 5 above)
Comment (nathanaelhoun) [reply CONTRIBUTING.md:35]: I think it's okey to link the documentation and code of conduct of the main repo: it will be easier to keep it in sync if there is only one copy.
For the website, it's not really important to pin the version, as it is for the other repositories. It's why it's not enforced in `package.json#engines`. As long as we stay on a recent version of node, NextJS should work.

File: CONTRIBUTING.md (hunk 6 above)
Comment (nathanaelhoun) [reply CONTRIBUTING.md:41]: That's right. In fact, I copy-pasted it [from the main repo](https://github.com/ferdium/ferdium-app/blob/16e66963f7907bacc42c189d41f3db0723864068/CONTRIBUTING.md#git), which as the same problem.

I'm wondering: is the `git` information really needed? If a developer is on GitHub and reads this file, doesn't he already know about git?

```suggestion
The version [2.23.0](https://git-scm.com/downloads) for Git is working fine for development. You can then use the console from Git to do the development procedure.
```

File: pages/faq.tsx (hunk 7 above)
Comment (nathanaelhoun) [reply pages/faq.tsx:110]: `next lint` asks for it, it's why the [latest commit deploy](https://github.com/ferdium/website/runs/6495657554?check_suite_focus=true) failed

File: CONTRIBUTING.md (hunk 4 above)
Comment (nathanaelhoun) [reply CONTRIBUTING.md:25]: See below

File: CONTRIBUTING.md (hunk 6 above)
Comment (vraravam) [reply CONTRIBUTING.md:41]: ok - we should change this in the main repo as well

File: pages/faq.tsx (hunk 7 above)
Comment (vraravam) [reply pages/faq.tsx:110]: oops

**Review by vraravam — APPROVED** [review]
LGTM
